docs/Using Fleet/Understanding-host-vitals.md

```diff
@@ -1,7 +1,7 @@
 <!-- DO NOT EDIT. This document is automatically generated. -->
 # Understanding host vitals
 
-Following is a summary of the detail queries hardcoded in Fleet used to populate the device details:
+The host vitals query collects the data displayed in the device inventory record for a host in Fleet.
```

Comment:
This file is automatically generated from source Go code, so you'll have to modify this and run `make generate-doc`:
https://github.com/fleetdm/fleet/blob/ff8c439d7303334ca4f52af24e1558094b40b6aa/server/service/osquery_utils/gen_queries_doc.go#L39

Reply:
Ah... I didn't see the large uppercase comment that said "DO NOT EDIT" 😅